dev/lerna.ts

```diff
@@ -0,0 +1,787 @@
+const SPLIT_CHAR = "END";
+
+const getPackages: Fig.Generator = {
+  script: "lerna ls",
+  postProcess: (output) =>
+    output.split("\n").map((packageName) => ({
```

Comment:
Can be simplified by using the same `name` variable name:
```suggestion
    output.split("\n").map(name => ({
```

Reply:
I agree with you that it makes it less code but I'm always struggling whether verbose code is better or *cleaner* code

Reply:
In this case, I feel like it's both cleaner and better code since it's easy to understand and easier to read.